src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -56,11 +58,17 @@ public void launchTask(final ExecutorDriver driver, final TaskInfo taskInfo) {
           .setState(TaskState.TASK_RUNNING)
           .build());
     } else if (taskInfo.getTaskId().getValue().contains(HDFSConstants.NAME_NODE_TASKID)) {
-      // Add the name node task and wait for activate message
       nameNodeTask = task;
+      driver.sendStatusUpdate(TaskStatus.newBuilder()
+          .setTaskId(nameNodeTask.taskInfo.getTaskId())
+          .setState(TaskState.TASK_RUNNING)
```

Comment:
True, we could consider this method as opposed to sending 2 task runnings with a different message. Without the init or bootstrap it's not functional, although you do need to kill a process...